etl/steps/data/garden/wb/2025-06-05/world_bank_pip.meta.yml

```diff
@@ -299,11 +310,11 @@ definitions:
     <%- endif %>
 
   poverty_line_subtitle: |-
-    <% if poverty_line in ["190", "215"] %>
+    <% if poverty_line in ["215", "300"] %>
```

Comment:
@Marigold Another question, is there any way to call these lists from lines 3, 4 and 5? It would be much better than repeating several times. I couldn't make it work